react/src/components/organisms/TabContainer/index.js

```diff
@@ -1,26 +1,63 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import shortid from 'shortid';
 import './style.css';
 import { TabContext } from './context';
 
 class TabContainer extends React.Component {
   constructor(props) {
     super(props);
+    // Allows Tab to interact directly with the tab body div through context.
+    this.tabBodyRef = React.createRef();
+    // When false, runs key down code for nested containers.
+    this.preventBodyKeyDown = false;
+    this.defaultContent = null;
+    this.useDefault = true;
+    const tabIds = new Map();
+    // This only works for class components because it is not re-generated on every render.
+    this.spanId = shortid.generate();
     this.setActiveTab = (tab, content = null) => {
       const state = {
         activeTab: tab
       };
       if (content) {
+        this.defaultContent = null;
         state.activeContent = content;
       }
       this.setState(state);
     };
+    this.focusOnTabBody = () => {
+      this.tabBodyRef.current.setAttribute('tabindex', '0');
```

Comment:
Instead of setting the body tabIndex to 0 here, is there a way to by default set the body tabIndex to 0? if it's a nested tab body or a tab body that has no nested tab?